src/plugins/plugin.gridlines.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+module.exports = function(Chart) {
+	var helpers = Chart.helpers;
+	var globalDefaults = Chart.defaults.global;
+
+	// Stores all gridLines that should be displayed
+	var lines = [];
+
+	function getGridLine(chart, scale, position, tickIndex) {
+		var chartArea = chart.chartArea;
+		var scaleOptions = scale.options;
+		var gridLines = scaleOptions.gridLines;
+
+		var lineWidth, lineColor, borderDash, borderDashOffset;
+
+		if (tickIndex === (typeof scale.zeroLineIndex !== 'undefined' ? scale.zeroLineIndex : 0)) {
```

Comment:
im gonna change this statement to `tickIndex !== undefined && tickIndex === zeroLineColor`, because i send undefined to tickIndex when getting the last extra gridLine when offsetGridLines is enabled...
[link](https://github.com/chartjs/Chart.js/pull/4117/files#diff-083085fb02710540ea967d175e159f86R104)

> 

>